hash/sgi_mips.xml

```diff
@@ -1548,54 +1548,80 @@ license:CC0
 		</part>
 	</software>
 
+	<software name="hotmix_5">
+		<description>Hot Mix 5</description>
+		<year>1993</year>
+		<publisher>Silicon Graphics</publisher>
+		<part name="cdrom" interface="cdrom">
+			<feature name="part_number" value="812-8101-005"/>
+			<!-- Origin: private dump -->
+			<diskarea name="cdrom">
+				<disk name="hot_mix_5" sha1="06e221fda5bb09c0ffcd23cbffa73cc1b503a980" />
+			</diskarea>
+		</part>
+	</software>
+
+	<software name="hotmix_7">
+		<description>Hot Mix 7</description>
+		<year>1994</year>
+		<publisher>Silicon Graphics</publisher>
+		<part name="cdrom" interface="cdrom">
+			<feature name="part_number" value="812-8101-007"/>
+			<!-- Origin: private dump -->
+			<diskarea name="cdrom">
+				<disk name="hot_mix_7" sha1="cd2af0d9d29c288c85c3308ec5c2a6cc25b698cc" />
+			</diskarea>
+		</part>
+	</software>
+
 	<software name="hotmix_8">
-		<description>Hot Mix Volume 8</description>
+		<description>Hot Mix 8</description>
```

Comment:
Please don't rename these unless you can verify the actual label on the disc. Hot Mix discs are not labeled consistently -- some are "Hot Mix X", some "Hot Mix Volume X". We should keep the same name that's used on the disc.

Reply:
It's a bit different, they are not even renamed consistently within one disc. They are called "HotMix X" on the CD itself, "Hot Mix - Volume X" on the sleeve's spine, and "HotMix - Volume X" in the booklet for example. I figured it would be easier to have one consistent naming scheme.

But if you want, I can revert that part of the PR, no problem

Reply:
Thanks! Yeah, it's a bit of a mess. I usually go by what's on the CDs themselves, as that seems to match what's on P/N and old price lists.